code/src/ServiceAPI/ApiServices.js

```diff
@@ -0,0 +1,66 @@
+let API_URL = 'https://happy-thoughts-technigo.herokuapp.com/thoughts'
+
+export const postThoughts = async (message) =>{
+  let messageSent
+  let error400
+
+  await fetch(API_URL, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json'
+    },
+    body: JSON.stringify({"message": message}) 
+  })
+  .then((response) =>{
+    if(response.status === 400){
+      error400 = true 
+      messageSent = false
+    }else{
+    return response.json()
+    }
+  })
+  .then((json)=>{
+    if(json){
+      messageSent = true
+    }
+    
+ })
+ .catch(err => messageSent= false)
+
+ return {messageSent, error400 };
+}
+
+export const fecthData = async () => {
```

Comment:
Remember to double check the spelling of function and variable names as to not confuse co-developers (also see line 52 below!)

Reply:
Thank you for your comments I would be attentive.